frontend/src/pages/article/components/article-view/article-view.tsx

```diff
@@ -53,6 +76,27 @@ const ArticleView: React.FC<Properties> = ({
         )}
         {!isShared && (
           <div className={styles.buttonsWrapper}>
+            {isArticleOwner && (
+              <>
+                <IconButton
+                  iconName="trashBin"
+                  className={styles.iconButton}
+                  iconClassName={styles.icon}
+                  onClick={handleDeleteArticle}
+                />
+                <RouterLink
+                  to={AppRoute.EDIT_ARTICLE.replace(':id', id as string)}
+                  state={article}
+                >
```

Comment:
Use common `Link` component,
And please change it on article-card component,
I missed it